docs/updating/updating_to_3.3.md

```diff
@@ -163,6 +163,14 @@ Check the Location ID of the "Components" Content item and set it as a value of
 If you are upgrading between Ibexa Commerce versions,
 add the `content/read` Policy with the Owner Limitation set to `self` to the "Ecommerce registered users" Role.
 
+### v3.3.2
+
+To update to v3.3.2, additionally run the following update script:
+
+``` sql
+ALTER TABLE ezpage_attributes MODIFY value LONGTEXT;
```

Comment:
In 3.3 we moved upgrade scripts to `vendor/ibexa/installer/upgrade/db/` so developers should run scripts like in 3.2:
`mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-3.3.1-to-3.3.2.sql`